src/components/tests/appbar.test.tsx

```diff
@@ -9,39 +9,69 @@ const mount = createMount();
 describe('AppBar', () => {
   let toggleShowOmniPanel: jest.Mock;
   let showSettings: jest.Mock;
+  let showTour: jest.Mock;
 
   beforeEach(() => {
     toggleShowOmniPanel = jest.fn();
     showSettings = jest.fn();
+    showTour = jest.fn();
   });
 
   test('renders correctly', () => {
     const root = mount(
-      <AppBar toggleShowOmniPanel={toggleShowOmniPanel} showSettings={showSettings} />,
+      <AppBar
+        toggleShowOmniPanel={toggleShowOmniPanel}
+        showSettings={showSettings}
+        showTour={showTour}
+      />,
     );
     expect(root).toMatchSnapshot();
   });
 
   test('toggles show omnipanel when dashboard button is clicked', () => {
     const root = mount(
-      <AppBar toggleShowOmniPanel={toggleShowOmniPanel} showSettings={showSettings} />,
+      <AppBar
```

Comment:
What do you think about creating a component generator function, since we are mounting the component with the same variables in every test?

```
 <AppBar
        toggleShowOmniPanel={toggleShowOmniPanel}
        showSettings={showSettings}
        showTour={showTour}
      />,
```

Something like `buildDoorWrapper` on `door.test.tsx`